### PR TITLE
DP-1.4- Added scheduler binding to forwarding group and the required interface ref config 

### PR DESCRIPTION
**Pull request by charantejag504**
DP-1.4- Added scheduler binding to forwarding group and the required interface ref config

### CODE REVIEW

File: feature/experimental/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go [hunk 1]
```diff
@@ -547,6 +547,11 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 		s := schedulerPolicy.GetOrCreateScheduler(tc.sequence)
 		s.SetSequence(tc.sequence)
 		s.SetPriority(tc.priority)
+		if dut.Vendor() == ondatra.JUNIPER {
+			t.Logf("Scheduler should be binded to a forwarding group")
+			Output := s.GetOrCreateOutput()
+			Output.SetOutputFwdGroup(tc.targetGroup)
+		}
```
Comment (sezhang2) [inline feature/experimental/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go:554]: Please add the tracking bug for this change.

**Review by sezhang2 — APPROVED** [review]
Please note, the vendor specific change is approved so that it does not block the traffic test. All the vendor specific change should be tracked by a bug.

**Comment by sachendras** [comment]
> Please note, the vendor specific change is approved so that it does not block the traffic test. All the vendor specific change should be tracked by a bug.

b/272376431 for Google internal communication

File: feature/experimental/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go (hunk 1 above)
Comment (rszarecki) [reply feature/experimental/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go:554]: As agreed, make it gated by deviation

**Pull request merged**